<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>651</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Support Reactions:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Max Bending Moment :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="maxBendingMomentLabel">
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Max Shear Force:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="maxShearForceLabel">
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Max Deflection:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="maxDeflectionLabel">
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="supportReactionsLabel">
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>651</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="FBDPlot_button">
      <property name="text">
       <string>FBD</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BMPlot_button">
      <property name="text">
       <string>BM Plot</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="SFPlot_button">
      <property name="text">
       <string>SF Plot</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="DeflectionPlotButton">
      <property name="text">
       <string>Deflection Plot</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="GenerateReport_button">
      <property name="text">
       <string>Generate Report</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Ok_button">
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
